CMakePCHCompiler.cmake

```diff
@@ -103,7 +103,7 @@ function(target_precompiled_header) # target [...] header
 			# /FI - force include of precompiled header
 			set(flags "/Yu${win_header} /Fp${win_pch} /FI${win_header}")
 		else()
-			set(flags "-include ${target_dir}/${header}")
+			set(flags "-include \"${target_dir}/${header}\"")
```

Comment:
Not sure about the reasoning for this change. Is it really necessary? Is there a problem is `target_dir` has spaces?

Reply:
YES EVERYTHING SUCH AS THIS IS ABSOLUTELY NECESSARY